### DISCUSSION TITLE
Having trouble synching Xbox Adaptive Controller with Virtual Boy

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#821] [General]
When I was talking to the people at Xbox they said they could only hold two devices the Xbox adaptive controller to hook up to and one has to be an Xbox and the other has to be a PC.

Considering the fact that I A) have a  Mac and B) have multiple consoles that run off the blue retro system as well as ones that run off the Brook system, is it safe to assume that I have to reset my Xbox adaptive controller every time I switch consoles on it?

Do I have to reset the blue retro device to forget the old because it considers each new instance a new controller it taps into?

Also retroonyx built the labels upside down, because I can't read which one is the boot button and which one is the reset button.

If I don't want to deal with council memory and just want to reset the Xbox adaptive controller and all the Blue Retros  each time I switch to a new system, may I have simple foolproof instructions if you just want to use your fight stick with your machine.

I noticed that there were different time intervals you have to hold down certain buttons and that confuses me and I don't know what to do.

Also consider the fact that my Macintosh can pick up blueretro.io when in Google Chrome mode.  I may want to make some adjustments there.

This is not a very intuitive system that you guys got unless you can explain it in one sheet on how you could just get to gaming on that one console if you don't care about other stuff.

**Comment by darthcloud** [comment]
You dont need to reset BlueRetro, just put the xbox adaptive in pairing mode again and the previous connection will be updated once the pairing is done.

**Reply by SinisterSticks** [reply]
May I have the correct procedure?  I was able to do it by holding the blueretro adapter's and Xbox Adaptive cobtroller's sync button until the "second change of state" of the light  on the Xbox Button.  

Got it to sync but got a different problem which will be posted in a new topic.
